core/src/com/unciv/logic/automation/NextTurnAutomation.kt

```diff
@@ -975,7 +975,10 @@ object NextTurnAutomation {
     data class CityDistance(val city1: CityInfo, val city2: CityInfo, val aerialDistance: Int)
 
     fun getClosestCities(civ1: CivilizationInfo, civ2: CivilizationInfo): CityDistance {
+        val city1 = CityInfo()
+        val city2 = CityInfo()
         val cityDistances = arrayListOf<CityDistance>()
+        cityDistances.add(CityDistance(city1, city2, Int.MAX_VALUE))
```

Comment:
You could just do `cityDistances += CityDistance(CityInfo(), CityInfo(), Int.MAX_VALUE)`.

I just checked through the code and all the places where the `city1` and `city2` fields are used check that both civs have > 0 cities so it _should_ be safe to use blank ones. Alternatively, just amending the if statement in the function that's causing the crash to check that both civs have at least 1 city could work too.